test/geocoders/geocodeapi.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from geopy import exc
+from geopy.geocoders import GeocodeAPI
+from geopy.point import Point
+
+from test.geocoders.util import BaseTestGeocoder, env
+
+
+@pytest.mark.skipif(
+    not bool(env.get('GEOCODEAPI_KEY')),
+    reason="No GEOCODEAPI_KEY env variable set"
+)
+class TestGeocodeAPI(BaseTestGeocoder):
+    @classmethod
+    def make_geocoder(cls, **kwargs):
+        return GeocodeAPI(env.get('GEOCODEAPI_KEY'), **kwargs)
```

Comment:
If `GEOCODEAPI_KEY` is mandatory for running these tests, it must be retrieved via `__getitem__` rather than with `get`: `env['GEOCODEAPI_KEY']`